### PR TITLE
Added Facebook tracking for user interactions

### PR DESCRIPTION
**Pull request by danthaman44**
Test plan:
-Verified tracking events are being sent via FB Pixel Helper Chrome extension
-Verified tracking events show up in Ads Manager
Screen Shot 2020-05-28 at 10 40 24 PM
Screen Shot 2020-05-28 at 10 09 29 PM

### CODE REVIEW

**Comment by jtmckibb** [comment]
wauuu

File: src/components/App/App.tsx [hunk 1]
```diff
@@ -29,6 +30,13 @@ const SellerPage = lazy(() => import('../SellerPage'));
 const MerchantsPage = lazy(() => import('../MerchantsPage'));
 const ErrorPage = lazy(() => import('../404Page'));
 
+const options = {
+  autoConfig: true, // set pixel's autoConfig
+  debug: true, // enable logs
+};
+const pixelId = '781939739005509';
```
Comment (jtmckibb) [inline src/components/App/App.tsx:37]: This isn't private right?

Comment (danthaman44) [reply src/components/App/App.tsx:37]: ahh shoot thats a good call, this needs to be private

Comment (danthaman44) [reply src/components/App/App.tsx]: Ok I changed it to be an env variable. Its probably best that we only set the env variable in prod, so we don't fire tracking events from local or staging.

**Pull request merged**